File: abc/abc153/abc153_c.cs
// https://atcoder.jp/contests/abc153/submissions/9749505

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var input = Console.ReadLine().Split().Select(long.Parse).ToArray();
            var N = input[0];
            var K = input[1];

            var H = Console.ReadLine().Split().Select(long.Parse).ToArray();
            Array.Sort(H);
            Array.Reverse(H);

            for (var i = 0; i < K; i++)
            {
                if (i >= H.Length)
                {
                    break;
                }
                H[i] = 0L;
            }

            Console.WriteLine(H.Sum());
        }
    }
}

File: abc/abc153/abc153_c.fs
// https://atcoder.jp/contests/abc153/submissions/11915085

let [| N; K |] = stdin.ReadLine().Split() |> Array.map int

stdin.ReadLine().Split()
|> Array.map int64
|> Array.sortDescending
|> Array.skip (min N K)
|> Array.sum
|> stdout.WriteLine
